recipe/meta.yaml

```diff
@@ -1,18 +1,18 @@
 {% set name = "pytest" %}
-{% set version = "7.1.2" %}
+{% set version = "7.3.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pytest-{{ version }}.tar.gz
-  sha256: a06a0425453864a270bc45e71f783330a7428defb4230fb5e6a731fde06ecd45
+  sha256: 434afafd78b1d78ed0addf160ad2b77a30d35d4bdf8af234fe621919d9ed15e3
 
 build:
   skip: true  # [py<37]
   number: 0
-  script: {{ PYTHON }} setup.py install --single-version-externally-managed --record record.txt
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation
```

Comment:
maybe add -vv

Reply:
Should that flag be included in the linter? I just followed the current suggested script to run.